cmd/serve_linux.go

```diff
@@ -55,6 +56,7 @@ var flagsServe = []cli.Flag{
 	altsrc.NewIntFlag(&cli.IntFlag{Name: "visitor-email-limit-burst", EnvVars: []string{"NTFY_VISITOR_EMAIL_LIMIT_BURST"}, Value: server.DefaultVisitorEmailLimitBurst, Usage: "initial limit of e-mails per visitor"}),
 	altsrc.NewDurationFlag(&cli.DurationFlag{Name: "visitor-email-limit-replenish", EnvVars: []string{"NTFY_VISITOR_EMAIL_LIMIT_REPLENISH"}, Value: server.DefaultVisitorEmailLimitReplenish, Usage: "interval at which burst limit is replenished (one per x)"}),
 	altsrc.NewBoolFlag(&cli.BoolFlag{Name: "behind-proxy", Aliases: []string{"P"}, EnvVars: []string{"NTFY_BEHIND_PROXY"}, Value: false, Usage: "if set, use X-Forwarded-For header to determine visitor IP address (for rate limiting)"}),
+	altsrc.NewBoolFlag(&cli.BoolFlag{Name: "disable-webui", Aliases: []string{"d"}, EnvVars: []string{"NTFY_DISABLE_WEBUI"}, Value: false, Usage: "if set, disables WebUI"}),
```

Comment:
Can you call this flag `disable-web`?

Reply:
I didn't realize the flag didn't need to have the same name as the variable.

Reply:
Actually. Darn. This is tricky. Because this is the server.yml file config option. I wanted to name it `web-enabled`, but boolean flags with a true default don't work. I don't know how to solve this.